src/components/editor/inputs/InputLang.jsx

```diff
@@ -96,12 +99,11 @@ const InputLang = (props) => {
               <input
                 type="radio"
                 className="form-check-input"
-                name="lang"
```

Comment:
get rid of the name="lang" to prevent the browser from trying to set the value of the radio button based on the grouping (we set based on our value in state)

Reply:
ah!  also nicer.